sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/factory/AbstractAzureServiceClientBuilderFactory.java

```diff
@@ -145,7 +145,6 @@ public T build() {
      */
     protected void configureCore(T builder) {
         configureApplicationId(builder);
-        configureAzureEnvironment(builder);
```

Comment:
does the Azure environment configuration not belong to sdk common configurations? it should be configured only for token credential client?

Reply:
configure env is more tricky, and I think it's better for us to call the set method directly on each builder, instead of configuring using the environment

Reply:
I see, and just notice that the javadoc in line 136 should be removed as well